src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -71,18 +71,31 @@ Namespace Microsoft.CodeAnalysis.Semantics
             Return arguments.ToImmutableAndFree()
         End Function
 
+        Private Function CreateConversion(expression As BoundExpression) As Conversion
+            If expression.Kind = BoundKind.Conversion Then
+                Dim conversion = DirectCast(expression, BoundConversion)
+                Dim method As MethodSymbol = Nothing
+                If conversion.Operand.Kind = BoundKind.UserDefinedConversion Then
```

Comment:
>conversion.Operand.Kind = BoundKind.UserDefinedConversion [](start = 19, length = 57)

It feels like we should also make sure that the conversion kind for the ```conversion``` node is user-defined conversion.